offline-tools/copyrighter/pkg/cmd/galasacopyrighter.go

```diff
@@ -335,12 +390,10 @@ func findNextNoneWhiteSpaceCharacter(input string, startIndexOfFollowingInput in
 }
 
 func commentNeedsNoChange(commentToCheck string) bool {
-	standardCopyright := `#
- # Copyright contributors to the Galasa project
- #
- # SPDX-License-Identifier: EPL-2.0
- #`
-	return commentToCheck == standardCopyright
+	if strings.Contains(commentToCheck, "# Copyright contributors to the Galasa project") && strings.Contains(commentToCheck, "# SPDX-License-Identifier: EPL-2.0") {
+		return true
+	}
+	return false
```

Comment:
Is this correct? Would the copyright comment be correct if it contained both of those strings or does it have to be the exact same as this to pass the OMP's scans:
```
# 
# Copyright contributors to the Galasa project
#
# SPDX-License-Identifier: EPL-2.0
#
```
If it needs to be the exact same, it be better passing in the actual copyright block we are looking for as a constant as you did before.

Reply:
good point, it has to be in that format. i think my initial code was too rigid, so if for example, the comment I was checking was exactly the same but had extra new lines or white spaces, the method would return false. 
But I have changed it so that the comment being checked is trimmed.